README-de-DE.md

```diff
@@ -0,0 +1,558 @@
+
+Dieses kleine Projekt begann damals im Main in Berlin bei der squantconf, wo
+ich mich damit beschäftigt habe, wie Perl Mönche ihre Dokumentation schreiben
+und mich ebenso über den Stand der READMEs im Node.js Ökosystem beschwert habe.
+Das hat mich dazu gebracht,
+[common-readme](https://github.com/noffle/common-readme) zu erstellen.
+Der Abschnitt zu "README Tipps" floss allerdings mit Tipps über, so dass ich
+mich entschied, diese in einem Artikel zu sammeln, wie READMEs geschrieben
+werden. So wurde die Kunst des README geboren!
+
+Du kannst mich unter `noffle@eight45.net` oder im Freenode IRC in `#eight45`
+erreichen.
+
+## Zum Weiterlesen
+
+- [README-Driven Development](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html)
+- [Documentation First](http://joeyh.name/blog/entry/documentation_first/)
+
+## Fußnoten
+
+1. <a name="footnote-1"></a>Danke,
+   [Sixes666](https://www.reddit.com/r/node/comments/55eto9/nodejs_the_art_of_readme/d8akpz6)!
+
+1. <a name="footnote-2"></a>Siehe [Jargon File](http://catb.org/~esr/jargon/html/R/README-file.html).
+   Die meisten Systeme heutzutage werden jedoch nicht Großbuchstaben vor
+   Kleinbuchstaben sortieren, so dass sich die Nützlichkeit der Konvention
+   auf die visuelle Hervorhebung durch die Nutzung der Großbuchstaben
+   beschränkt.
+
+## Danksagung
+
+Ein inniger Dank an dich, [@mafintosh](https://github.com/mafintosh) und
+[@feross](https://github.com/feross) für die Ermunterung, die ich brauchte, um
+diese Idee umzusetzen und anzufangen zu schreiben!
+
+Danke auch an die folgenden großartigen Leser, die Fehler bemerkt und mir PRs
+gesendet haben :heart: :
+
+- [@ungoldman](https://github.com/ungoldman)
+- [@boidolr](https://github.com/boidolr)
+- [@imjoehaines](https://github.com/imjoehaines)
+- [@radarhere](https://github.com/radarhere)
+- [@joshmanders](https://github.com/joshmanders)
+- [@ddbeck](https://github.com/ddbeck)
+- [@RichardLitt](https://github.com/RichardLitt)
+- [@StevenMaude](https://github.com/StevenMaude)
+- [@KrishMunot](https://github.com/KrishMunot)
+- [@chesterhow](https://github.com/chesterhow)
+- [@sjsyrek](https://github.com/sjsyrek)
+- [@thenickcox](https://github.com/thenickcox)
+
+Dank an [@qihaiyan](https://github.com/qihaiyan) für das Übersetzen der Kunst
+des README ins Chinesische! Die folgenden Nutzer haben auch Beiträge
+eingebracht:
+
+- [@BrettDong](https://github.com/brettdong) für das Revidieren der Punktierung
+  in der chinesischen Version.
+- [@Alex-fun](https://github.com/Alex-fun)
+- [@HmyBmny](https://github.com/HmyBmny)
+- [@vra](https://github.com/vra)
+
+Dank an [@lennonjesus](https://github.com/lennonjesus) für das Übersetzen der
+Kundes des README in Brasilianisches Portugiesisch! Die folgenden Nutzer haben
+auch Beiträge eingebracht:
+
+- [@rectius](https://github.com/rectius)
+
+Dank an [@jabiinfante](https://github.com/jabiinfante) für das Übersetzen der
+Kunst des README ins Spanisch!
+
```

Comment:
Also, add your contribution to the other files (at least the main English one)

Reply:
I'd defer this honor to @noffle as a separate commit, which updates all files at once.
I'd highlight your name as well, since you reviewed the contribution :-)